backend/typescript/services/implementations/camperService.ts

```diff
@@ -46,6 +62,32 @@ class CamperService implements ICamperService {
               `Camp session ${campers[0].campSession} not found.`,
             );
           }
+          let totalPayment = 0
+          let campFees = 0
+          let earlyDropoffAndLatePickUp = 0
+          let specialNeedsCamper = {camperName: "", campName: "", sessionDates: "", registrantName: "", registrantEmail: "", registrantPhoneNumber: "", specialNeeds: ""}
+          let emailServiceCampers : any[] = [];
+          let camp = await MgCamp.findById(existingCampSession.camp)
+          newCampers.map(camper => {
+            campFees += camper.charges.camp
+            earlyDropoffAndLatePickUp += camper.charges.earlyDropoff + camper.charges.latePickup
+            if(camper.specialNeeds && camp) {
+              specialNeedsCamper = {camperName: camper.firstName + " "+camper.lastName, campName: camp.name, sessionDates: sessionDatesToString(existingCampSession?.dates), registrantName: camper.contacts[0].firstName + " "+camper.contacts[0].lastName, registrantEmail: camper.contacts[0].email, registrantPhoneNumber: camper.contacts[0].phoneNumber, specialNeeds: camper.specialNeeds}
```

Comment:
we should try to use template strings whenever we can, looks a bit cleaner than adding strings together. for example instead of `camper.firstName + " " + camper.lastName` we'd use `` `${camper.firstName} ${camper.lastName}` ``